Cargo.toml

```diff
@@ -80,6 +81,7 @@ ego-tree = "0.6.2"
 
 serde_json = "1.0.122"
 serde = "1.0.204"
+wasm-bindgen = "0.2.93"
```

Comment:
why is `wasm-bindgen` now added as a general dependency in addition to a wasm only dependency below? it should still only be necessary when compiling to wasm